AutomatedTesting/Gem/PythonTests/atom_renderer/test_Atom_GPUTests.py

```diff
@@ -40,6 +43,34 @@ def golden_images_directory():
     return golden_images_dir
 
 
+def create_zip_archive(archive_path):
```

Comment:
I think this could be more explicit, as it does only archive screenshots
def create_screenshots_archive(source_path_of_screenshots):